config.py

```diff
@@ -1,20 +1,27 @@
+# All configuration listed here is defaut
+# parameters. You can override them in
```

Comment:
Could  be: 'All configuration parameters defined in this file can be overriden in instance/config.py`